rskj-core/src/main/java/org/ethereum/core/BlockFactory.java

```diff
@@ -189,46 +184,33 @@ public BlockHeader decodeHeader(RLPList rlpHeader, boolean sealed) {
         Coin paidFees = RLP.parseCoin(rlpHeader.get(13).getRLPData());
         Coin minimumGasPrice = RLP.parseSignedCoinNonNullZero(rlpHeader.get(14).getRLPData());
 
-        int headerElementsParsed = 15;
+        int r = 15;
 
         int uncleCount = 0;
-        if (rlpHeader.size() == RLP_HEADER_SIZE_WITH_MERGED_MINING_AND_FORK_DATA ||
-                rlpHeader.size() == RLP_HEADER_SIZE_WITH_MERGED_MINING ||
-                rlpHeader.size() == RLP_HEADER_SIZE_WITH_FORK_DATA ||
-                rlpHeader.size() == RLP_HEADER_SIZE) {
-            byte[] ucBytes = rlpHeader.get(headerElementsParsed++).getRLPData();
+        if ((rlpHeader.size() == 19) || (rlpHeader.size() == 16)) {
```

Comment:
Can we add some name to the numbers 16 and 19 ?

Reply:
Sure, I lost that on a recent rollback. Fixed!